### PR TITLE
Filter unnecessary data for product attribute metadata

### PR DESCRIPTION
**Pull request by melnikovi**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: AggregatedServices/Api/Data/AttributeInformationInterface.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+     */
+    public function setLabel($label);
+
+    /**
+     * @return string
+     */
+    public function getLabel();
+
+    /**
+     * Set values
```
Comment (paliarush) [inline AggregatedServices/Api/Data/AttributeInformationInterface.php]: Should it be `Set options`?

File: AggregatedServices/Model/Service/GetCart.php [hunk 2]
```diff
@@ -164,11 +182,35 @@ public function get(
             ->create();
         $aggregatedCart->setProducts($this->productRepository->getList($productsSearchCriteria));
 
-        // Fetch cart products
+        // Fetch attributes metadata
         if ($productAttributesSearchCriteria) {
```
Comment (paliarush) [inline AggregatedServices/Model/Service/GetCart.php:187]: Let's at least move it to a separate method to make code cleaner

**Review by paliarush — APPROVED** [review]

File: AggregatedServices/Model/Service/GetCart.php (hunk 2 above)
Comment (melnikovi) [reply AggregatedServices/Model/Service/GetCart.php:187]: Resolved

File: AggregatedServices/Api/Data/AttributeInformationInterface.php (hunk 1 above)
Comment (melnikovi) [reply AggregatedServices/Api/Data/AttributeInformationInterface.php]: Resolved

**Comment by melnikovi** [comment]
http://10.234.214.130:8080/job/WebAPI-Tests/946/

**Review by paliarush — APPROVED** [review]

**Pull request merged**